src/main/java/se/sundsvall/notes/api/FindRevisionsResponse.java

```diff
@@ -0,0 +1,61 @@
+package se.sundsvall.notes.api;
```

Comment:
If this file should be kept, it should be kept in `src/main/java/se/sundsvall/notes/api/model`, right?